src/Fhir.Anonymizer.Shared.AzureDataFactoryPipeline/scripts/CustomActivity.ps1

```diff
@@ -0,0 +1,19 @@
+<#
+.PARAMETER AppVersion
+    Specify the version of app.
+#>
+
+[cmdletbinding()]
+param(
+    [string]$AppVersion
+)
+
+# Script to run custom activity executable
+$AppFolder = "$AppVersion.AdfApplication"
```

Comment:
Could we use same app name for all version exe?

Reply:
Using the same name might be confusing after uploaded to blobs?

Reply:
Not sure what the path will be in the blob? It might be better if users can upload both R4 and STU 3 ADF binaries without overwriting each other.